Formula/k3d.rb

```diff
@@ -18,18 +18,19 @@ class K3d < Formula
   def install
     system "go", "build",
            "-mod", "vendor",
-           "-ldflags", "-s -w -X github.com/rancher/k3d/version.Version=v#{version}",
+           "-ldflags", "-s -w -X github.com/rancher/k3d/v3/version.Version=v#{version}"\
+           " -X github.com/rancher/k3d/v3/version.K3sVersion=latest",
```

Comment:
what is the difference btw `Version` and `K3sVersion`?

Reply:
Version is the k3d CLI version and k3sVersion is the default k3s version used when creating a new cluster with k3d 👍